src/constants/GlobalConstants.js

```diff
@@ -105,3 +105,16 @@ export const EXPLORER_URL = {
 export const TIME_TOAST_ANIMATION = 5000;
 export const DELAY_REMOVE_CONTRACT = 1000;
 export const TIME_REMOVE_CONTRACT = TIME_TOAST_ANIMATION + DELAY_REMOVE_CONTRACT;
+
+export const ERC20_HASHES = {
+	'allowance(address,address)': 'dd62ed3e',
+	'approve(address,uint256)': '095ea7b3',
```

Comment:
You need to check the ERC20 interface: https://theethereum.wiki/w/index.php/ERC20_Token_Standard